lib/v2/8264/list.js

```diff
@@ -0,0 +1,92 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const timezone = require('@/utils/timezone');
+const { parseDate, parseRelativeDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const { id = '751' } = ctx.params;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 30;
+
+    const rootUrl = 'https://www.8264.com';
+    const currentUrl = new URL(`list/${id}`, rootUrl).href;
+
+    const { data: response } = await got(currentUrl, {
+        responseType: 'buffer',
+    });
+
+    const $ = cheerio.load(iconv.decode(response, 'gbk'));
+
+    $('div.newslist_info').remove();
+
+    let items = $('div.newlist_r, div.newslist_r, div.bbslistone_name, dt')
+        .find('a')
+        .slice(0, limit)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const link = item.prop('href');
+
+            return {
+                title: item.text(),
+                link: /^http/.test(link) ? link : new URL(link, rootUrl).href,
+            };
+        });
+
+    items = await Promise.all(
+        items.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: detailResponse } = await got(item.link, {
+                    responseType: 'buffer',
+                });
+
+                const content = cheerio.load(iconv.decode(detailResponse, 'gbk'));
+
+                content('a.syq, a.xlsj, a.titleoverflow200, #fjump').remove();
+                content('i.pstatus').remove();
+                content('div.crly').remove();
+
+                const pubDate = content('span.pub-time').text() || content('span.fby').text().split('发表于').pop().trim();
```

Comment:
`$('span.fby span').attr('title')` seems better than relative time.
The inner `span` only appears when relative time is shown on webpage.
https://bbs.8264.com/thread-5712304-1-1.html

Reply:
Thank you for the tip. 😊